changelog/1.38.0.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "1.38.0"
+description: "Released on 01/04/2023"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.38.0.
+
+### Features ✨
+
+- Individual image tags can now be decommissioned. Existing workspaces using a
+  decommissioned tag will be migrated to use the default tag upon rebuild. Adds
+  new organization-level roles: Organization Super Manager, Organization Image
+  Manager, and Organization Importer. These roles allow finer-grained access
+  control around image and registry management. For more information, see
+  Organization Roles.
+  > ℹ️ ️Migration: existing users with the Organization Manager role will be
+  > migrated to Organization Super Manager, and users with the Organization
+  > Member role will be migrated to Importer. Both of these changes will result
+  > in no effective permission changes for existing users.
+- Improves web terminal reconnection by leveraging GNU Screen if available
+  inside the workspace. Workspaces without screen installed will no longer
+  support reconnection.
+
+### Bug fixes 🐛
+
+- Fixed an issue where operations on API keys were not audit-logged.
```

Comment:
This is _kinda_ true. The endpoints related to api keys are audited in this release, but not all operations on API keys are.

Reply:
Ack. I'll omit it then for now to avoid confusion.